components/org.wso2.carbon.identity.oauth.scope.endpoint/src/test/java/org/wso2/carbon/identity/oauth/scope/endpoint/util/ScopeUtilsTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.scope.endpoint.util;
+
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.oauth.scope.endpoint.dto.ErrorDTO;
+import org.wso2.carbon.identity.oauth.scope.endpoint.dto.ScopeDTO;
+import org.wso2.carbon.identity.oauth.scope.endpoint.dto.ScopeToUpdateDTO;
+import org.wso2.carbon.identity.oauth2.bean.Scope;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.mockito.Mockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+
+public class ScopeUtilsTest {
+
+
+    @Test(description = "Testing getErrorDTO")
+    public void testGetErrorDTO() throws Exception {
+
+        ErrorDTO errorDTOexpected = new ErrorDTO();
+        errorDTOexpected.setCode("AJYRKLWB68NSB9");
+        errorDTOexpected.setMessage("Lifecycle exception occurred");
+        errorDTOexpected.setDescription("Error occurred while changing lifecycle state");
+
+        ErrorDTO errorDTO1 = ScopeUtils.getErrorDTO("Lifecycle exception occurred", "AJYRKLWB68NSB9", "Error occurred while changing lifecycle state");
+        Assert.assertEquals(errorDTO1.getCode(), errorDTOexpected.getCode());
+        Assert.assertEquals(errorDTO1.getMessage(), errorDTOexpected.getMessage());
+        Assert.assertEquals(errorDTO1.getDescription(), errorDTOexpected.getDescription());
+
+    }
+
+    @Test(description = "Testing getScope")
+    public void testGetScope() throws Exception {
+
+        ScopeDTO scopeDTO = mock(ScopeDTO.class);
+        ArrayList<String> scope = new ArrayList<>();
+
+        when(scopeDTO.getBindings()).thenReturn(scope);
+        when(scopeDTO.getName()).thenReturn("Lifecycle exception occurred");
+        when(scopeDTO.getDescription()).thenReturn("Error occurred while changing lifecycle state");
+
+        Scope scopeexpected = new Scope("Error occurred while changing lifecycle state", "Lifecycle exception occurred", new ArrayList<String>(Arrays.asList("scope1", "scope2")));
+        scopeexpected.setName("Lifecycle exception occurred");
+        scopeexpected.setDescription("Error occurred while changing lifecycle state");
+        scopeexpected.setBindings(scope);
+
+        Scope scope1 = ScopeUtils.getScope(scopeDTO);
```

Comment:
I think you don't need to build an expected scope here. Instead you can directly use above strings which are used to build expected scope for assertions.